signer/src/main.rs

```diff
@@ -74,11 +75,18 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         revision = signer::GIT_COMMIT,
         arch = signer::TARGET_ARCH,
         env_abi = signer::TARGET_ENV_ABI,
+        signer_version = signer::VERSION,
```

Comment:
I don't think we are actively using the package version (I guess `revision` is already enough)

Reply:
Yeah, I think we should probably remove it, only because that "useless info" may be confusing to someone. Right now it will always say `0.0.1` (even though we're at `0.0.9-rc6`).

This is more of a flaw in our build process -- there is tooling to handle version bumps and getting this properly set, but we're not using any of it atm.

So, if/until we've got that implemented, I would leave this out, just to avoid the aforementioned confusion.